rtrlib/transport/tcp/tcp_transport.c

```diff
@@ -48,6 +48,7 @@ int tr_tcp_open(void *tr_socket)
 {
 	int rtval = TR_ERROR;
 	struct tr_tcp_socket *tcp_socket = tr_socket;
+	const struct tr_tcp_config *config = &tcp_socket->config;
```

Comment:
I'm not against adding a variable as a shortcut, but right now the use in this function is inconsistent `tcp_socket->config` is still used in several places.
Keeping the `config` variable in this PR would be fine by me, but I would create a separate commit for it's addition and use in the function.